internal/checker/terraform/rules/gcp/google_folder_iam_binding_manage_service_acc_restricted.rego

```diff
@@ -0,0 +1,58 @@
+# METADATA
+# title: "Prevent the use of roles that grant the ability to assume and manage all service accounts within a folder-level context"
+# description: "Implement a restriction to disallow the utilization of roles that provide the capability to impersonate and oversee all service accounts at the folder level, enhancing security in this context."
+# scope: package
+# related_resources:
+# - https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/google_folder_iam#google_folder_iam_binding
+# custom:
+#   id: CB_TFGCP_006
+#   severity: CRITICAL
+package lib.terraform.CB_TFGCP_006
+
+import future.keywords.in
+
+isvalid(block) {
+	supported_resources := ["google_folder_iam_binding", "google_folder_iam_member"]
+	block.Type == "resource"
+	some label in block.Labels
+	label in supported_resources
+}
+
+resource[resource] {
+	some block in pass
+	resource := concat(".", block.Labels)
+}
+
+resource[resource] {
+	some block in fail
+	resource := concat(".", block.Labels)
+}
+
+pass[block] {
+some block in input
+isvalid(block)
+invalid_roles := ["roles/iam.securityAdmin", "roles/iam.serviceAccountAdmin", "roles/iam.serviceAccountKeyAdmin", "roles/iam.serviceAccountUser", "roles/iam.serviceAccountTokenCreator", "roles/iam.workloadIdentityUser", "roles/dataproc.editor", "roles/dataproc.admin", "roles/dataflow.developer", "roles/resourcemanager.folderAdmin", "roles/resourcemanager.folderIamAdmin", "roles/resourcemanager.projectIamAdmin", "roles/resourcemanager.organizationAdmin", "roles/cloudasset.viewer", "roles/cloudasset.owner", "roles/serverless.serviceAgent", "roles/dataproc.serviceAgent"]
```

Comment:
fix indention and add new line in entry of array

invalid_roles := ["roles/iam.securityAdmin", 
                             "roles/iam.serviceAccountAdmin",
                             "etc]